aiida/engine/processes/exit_code.py

```diff
@@ -45,7 +48,11 @@ def __call__(self, identifier):
 
         :param identifier: the identifier of the exit code. If the type is integer, it will be interpreted as
             the exit code status, otherwise it be interpreted as the exit code label
+        :type identifier: str
+
         :returns: an ExitCode named tuple
+        :rtype: (int, str)
```

Comment:
Shouldn't this be `:class:aiida.engine.ExitCode` even though it is some sort of a tuple?